source/icu.net.tests/ResourceBundleTests.cs

```diff
@@ -51,14 +51,16 @@ public void GetStringContentsWithKeys()
 			}
 		}
 
-		[TestCase("en_US", ExpectedResult = "[a b c d e f g h i j k l m n o p q r s t u v w x y z]")]
-		[TestCase("de_DE", ExpectedResult = "[a ä b c d e f g h i j k l m n o ö p q r s ß t u ü v w x y z]")]
-		[TestCase("fr_FR", ExpectedResult = "[a à â æ b c ç d e é è ê ë f g h i î ï j k l m n o ô œ p q r s t u ù û ü v w x y ÿ z]")]
+		[TestCase("en_US", ExpectedResult = "[abcdefghijklmnopqrstuvwxyz]")]
+		[TestCase("de_DE", ExpectedResult = "[aäbcdefghijklmnoöpqrsßtuüvwxyz]")]
+		[TestCase("fr_FR", ExpectedResult = "[aàâæbcçdeéèêëfghiîïjklmnoôœpqrstuùûüvwxyÿz]")]
```

Comment:
this is locale data so of course it could change or be reformatted. 

If you have ICU here, you could actually parse it into a UnicodeSet and do a comparison there.

Reply:
substring containment might be a quicker option.. i.e. make sure that english contains a, b, z,  German contains `ß` etc.